app/validators/application_fulfilment_validator.rb

```diff
@@ -1,24 +1,6 @@
-class ApplicationFulfilmentValidator < ActiveModel::Validator
-  include Routing
-
-  attr_reader :record
-
-  def validate(record)
-    @record = record
-
-    errors = perform_validations
-    add_errors(errors)
-  end
-
-  # Used by the `Routing` module to build the urls
-  def default_url_options
-    { id: record }
-  end
-
+class ApplicationFulfilmentValidator < BaseFulfilmentValidator
   private
 
-  # More validations can be added here
-  # Errors, when more than one, will maintain the order
   def perform_validations
```

Comment:
@hiboabd Just a minor suggestion. What do you think of changing method name from `perform_validations` to `validation_errors` as this method returns collection of errors ?

Reply:
Thanks, this got merged so too late to do now 😄 it's doing both so both names are accurate. I'm happy to leave as is for now